src/OrchardCore/OrchardCore.Abstractions/Json/Dynamic/DefaultJsonDyanmicValueHandler.cs

```diff
@@ -0,0 +1,47 @@
+using System;
+using System.Collections.Generic;
+using System.Text.Json;
+using System.Text.Json.Nodes;
+using Json.More;
+
+namespace OrchardCore.Json.Dynamic;
+public class DefaultJsonDyanmicValueHandler : IJsonDynamicValueHandler
+{
+    public bool GetValue(JsonObject jsonObject, Dictionary<string, object> dynamicValueDict, string memberName, JsonNode currentNode)
+    {
+        if (currentNode is JsonValue jsonValue)
+        {
+            var valueKind = jsonValue.GetValueKind();
+            switch (valueKind)
+            {
+                case JsonValueKind.String:
+                    if (memberName == "Value")
```

Comment:
If I remember correctly the only two types of fields that currently take values from strings in OC seem to be these two types, which are always wrapped in Value